packages/ui-components/src/icon/widgets/tabbarsvg.ts

```diff
@@ -20,6 +20,7 @@ export class TabBarSvg<T> extends TabBar<T> {
   constructor(options: TabBar.IOptions<T> = {}) {
     options.renderer = options.renderer || TabBarSvg.defaultRenderer;
     super(options);
+    this.addButtonNode.innerHTML = addIcon.svgstr;
```

Comment:
You should prefer the `.element` method:

```ts
addIcon.element({ container: this.addButtonNode });
```

> see [documentation](https://jupyterlab.readthedocs.io/en/stable/extension/ui_components.html#how-to-render-an-icon-into-a-dom-node)

Reply:
thanks, I was pretty sure there was a better way